packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```diff
@@ -851,6 +852,20 @@
             "deletedfile": "deleted file: __file__",
             "appendedfile": "appended to file: __file__"
         },
+        "encoding": {
+            "none": "None",
```

Comment:
Value may be none - but do we really mean "no encoding" ? I presume this is the "old" behaviour where we try to guess binary or not ? Should the label be "auto" ?  (or does that then imply we try to guess based on the machine language settings etc...  - I don't know - just asking..)

Reply:
Agreed - that's a great list of encodings, but we need to be clearer what the default selection means.

Reply:
I think current (old) implementation do not automatically guess encoding.
It expects utf8 string when read/write files as follows:
- [write] string -> Buffer[utf8:default] -> file
- [read] file -> Buffer -> toString[utf8:default]

`utf8[default]` label appropriate instead of `none`?

Reply:
Sounds good

Reply:
Thank you. Pushed update.